app/controllers/dmails_controller.rb

```diff
@@ -1,13 +1,14 @@
 class DmailsController < ApplicationController
   respond_to :html
-  respond_to :json, only: %i[index show destroy mark_as_read mark_all_as_read]
+  respond_to :json, except: %i[new create]
   before_action :member_only
+  before_action :load_dmail, except: %i[index new create mark_all_as_read]
```

Comment:
Really not a fan of this pattern. Just stick to how it was previously.

Reply:
Done